src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -217,27 +244,36 @@ private void SolutionEvents_AfterClosing()
             // Initialize if not already done.
             await InitializeAsync();
 
-            if (_solutionManager.Value.IsSolutionFullyLoaded)
-            {
-                // start background runner if not yet started
-                // ignore the value
-                var ignore = _backgroundJobRunner.Value;
-            }
-
             var pendingRestore = _pendingRestore;
 
             // on-board request onto pending restore operation
             _pendingRequests.Value.TryAdd(request);
 
-            // Await completion of the requested restore operation.
-            // The caller will be unblocked immediately upon
-            // cancellation request via provided token.
-            // Method returns false in case of cancellation is requested.
-            var cancellationTcs = new TaskCompletionSource<bool>();
-            using (token.Register(() => cancellationTcs.SetResult(false)))
-            using (_joinableCollection.Join())
+            Logger.LogInformation($"Scheduled restore request {pendingRestore}.");
```

Comment:
@alpaix : I thought you told me that was the case and I took your word for it! 😕